### PR TITLE
Update Languages/es.xml

### PR DESCRIPTION
**Pull request by Jhovany200**
minor corrections and new translations.

### CODE REVIEW

File: Languages/es.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-Este es el recurso de traducción en español para Dism ++.
+Este es el recurso de traducción en español para Dism++.
```
Comment (mingkuang-Chuyu) [inline Languages/es.xml]: Very careful.

File: Languages/es.xml [hunk 2]
```diff
@@ -91,7 +91,7 @@ Gracias a: Oleada, Jhovany200, nomemolesteis@gmail.com por la traducción.
     <String Target="重启进入RE并为我启动Dism++" Link="Reiniciar para WinRE y ejecutar Dism++"/>
     <String Target="启动RE环境" Link="Iniciar WinRE"/>
     <String Target="添加删除Dism插件" Link="Añadir o eliminar complemento Dism++"/>
-    <String Target="在官方网站可以获取帮助文档，并提出你的宝贵意见" Link="La documentación está disponible en “www.chuyu.me”. ¡Agradecemos tus comentarios!"/>
+    <String Target="在官方网站可以获取帮助文档，并提出你的宝贵意见" Link="La documentación está disponible en &quot;www.chuyu.me&quot;. ¡Agradecemos tus comentarios!"/>
```
Comment (mingkuang-Chuyu) [inline Languages/es.xml]: “  or  &quot;， You can give priority to using the local habits. I do not know what you are used to.
: )

**Comment by mingkuang-Chuyu** [comment]
Do you have any other changes? If not, I will merge this.

**Comment by Jhovany200** [comment]
merge pull requests.

**Comment by mingkuang-Chuyu** [comment]
Thanks

**Pull request merged**